controllers/renovate_util.go

```diff
@@ -29,33 +30,54 @@ import (
 )
 
 const (
-	RenovateImageEnvName    = "RENOVATE_IMAGE"
-	DefaultRenovateImageUrl = "quay.io/konflux-ci/mintmaker-renovate-image:cdbc220"
-	DefaultRenovateUser     = "red-hat-konflux"
-	CaConfigMapLabel        = "config.openshift.io/inject-trusted-cabundle"
-	CaConfigMapKey          = "ca-bundle.crt"
-	CaFilePath              = "tls-ca-bundle.pem"
-	CaMountPath             = "/etc/pki/ca-trust/extracted/pem"
-	CaVolumeMountName       = "trusted-ca"
+	RenovateImageEnvName                    = "RENOVATE_IMAGE"
+	DefaultRenovateImageUrl                 = "quay.io/konflux-ci/mintmaker-renovate-image:083371d"
+	DefaultRenovateUser                     = "red-hat-konflux"
+	CaConfigMapLabel                        = "config.openshift.io/inject-trusted-cabundle"
+	CaConfigMapKey                          = "ca-bundle.crt"
+	CaFilePath                              = "tls-ca-bundle.pem"
+	CaMountPath                             = "/etc/pki/ca-trust/extracted/pem"
+	CaVolumeMountName                       = "trusted-ca"
+	BranchPrefix                            = "konflux/component-updates/"
+	NamespaceWideRenovateConfigName         = "namespace-wide-nudging-renovate-config"
+	CustomRenovateConfigMapAnnotation       = "build.appstudio.openshift.io/custom_renovate_config_map"
```

Comment:
It's used only for nudging,  What about `nudge_renovate_config_map` or something similar?